src/components/text-editor/text-editor.stories.mdx

```diff
@@ -157,28 +83,8 @@ will prevent any input that would cause the content length to exceed it.
 <Canvas>
   <Story
     name="with optional character limit"
-    parameters={{ chromatic: { disableSnapshot: true } }}
-  >
-    {() => {
-      const [value, setValue] = useState(EditorState.createEmpty());
-      const limit = 100;
-      const ref = useRef(null);
-      return (
-        <div style={{ padding: "4px" }}>
-          <TextEditor
-            onChange={(newValue) => {
-              setValue(newValue);
-            }}
-            value={value}
-            ref={ref}
-            onSubmit={() => {}}
-            labelText="Text Editor Label"
-            characterLimit={limit}
-          />
-        </div>
-      );
-    }}
-  </Story>
+    story={stories.WithUserDefinedCharacterLimit}
```

Comment:
Why we are changing this from the name "with optional character limit" to "WithUserDefinedCharacterLimit'"?

Reply:
I personally don't like `WithUserDefinedCharacterLimit`, as a name, as really the user is not defining anything, but the consumer is. This applies to the exported story from `text-editor-stories.tsx` and the heading name within this markdown file (I know it was already there before the refactor). I think a name like `optionalCharacterLimit` or just `characterLimit` would be a good name for the exported story, and perhaps change the header to just `with a character limit`. Again, this is my personal preference, so feel free to disagree and not change its not super necessary 👍